src/app/portlets/dot-service-integration/dot-service-integration-configuration/dot-service-integration-configuration-list/dot-service-integration-configuration-list.component.html

```diff
@@ -0,0 +1,37 @@
+<div class="dot-service-integration-configuration-list__results">
+    <div
+        class="dot-service-integration-configuration-list__item"
+        *ngFor="let site of siteConfigurations; let i = index"
+    >
+        <div class="dot-service-integration-configuration-list__name">
+            {{ site.name }}
+        </div>
+        <div class="dot-service-integration-configuration-list__host-key">
+            {{ messagesKey['service.integration.key'] }} {{ site.id }}
+        </div>
+        <dot-icon-button
```

Comment:
Two `*ngIf="site.configured"`, you can group, in fact this looks like a if/else.